src/forms/assetForm.tsx

```diff
@@ -82,13 +82,13 @@ export interface AssetFormProps {
   isOpen: boolean;
   submitForm(Asset: ShallowAssetObject, headers: any): Promise<any> | void;
   pageScroll?(): void;
-  datacenters: Array<DatacenterObject>;
-  currDatacenter: DatacenterObject;
+  currSite: DatacenterObject;
   changePlan: ChangePlan;
 }
 interface AssetFormState {
   values: AssetObject;
-  currDatacenter?: DatacenterObject;
+  currSite?: DatacenterObject;
+  sites: Array<DatacenterObject>;
```

Comment:
`sites` was moved to state and it's value is now determined inside of AssetForm, instead of being passed in